Sources/StreamChat/Controllers/CurrentUserController/CurrentUserController.swift

```diff
@@ -256,6 +256,18 @@ public extension CurrentChatUserController {
             }
         }
     }
+    
+    /// Deletes all the local downloads of file attachments.
+    ///
+    /// - Parameter completion: Called when files have been deleted or when an error occured.
+    func deleteAllLocalAttachmentDownloads(completion: ((Error?) -> Void)? = nil) {
```

Comment:
should we call this on logout as you suggested?

Reply:
I went for direct flush of the directory because logout deletes DTOs and the delete all operates on the DB. Less updating then when logging out.